tools/docker/builder/openwrt/Dockerfile

```
@@ -92,7 +92,8 @@ RUN cp feeds.conf.default feeds.conf \
     && if [ "$TARGET_PROFILE" = "DEVICE_NETGEAR_RAX40" ] ; then \
         grep -v -E 'CONFIG_SDK|CONFIG_MAKE_TOOLCHAIN' feeds/intel/rax40.profile >> .config ;\
     fi ;\
-    make defconfig \
+    rm -rf tmp \
```

Adding an explanatory comment here would be nice. A little trick to add a comment:
```suggestion
    : "Installing intel feed doesn't correctly regenerate kernel .package-info"; \
    : "force regeneration by removing it"; \
    rm -rf tmp \
```

---

Nice! There were multiple other occasions where I wanted to put comments in the middle like here, but I couldn't think of any way to do it.

I'll do this one here and remember it for the next time.

Thanks!